packages/service-library/src/servicelib/long_running_tasks/_rabbit/lrt_client.py

```diff
@@ -0,0 +1,134 @@
+import logging
+from datetime import timedelta
+from typing import Any, Final
+
+from models_library.rabbitmq_basic_types import RPCMethodName
+from pydantic import PositiveInt, TypeAdapter
+
+from ...logging_utils import log_decorator
+from ...long_running_tasks.task import RegisteredTaskName
+from ...rabbitmq._client_rpc import RabbitMQRPCClient
+from .._serialization import string_to_object
+from ..models import RabbitNamespace, TaskBase, TaskContext, TaskId, TaskStatus
+from ._models import RPCErrorResponse
+from .namespace import get_namespace
+
+_logger = logging.getLogger(__name__)
+
+_RPC_TIMEOUT_VERY_LONG_REQUEST: Final[PositiveInt] = int(
+    timedelta(minutes=60).total_seconds()
+)
+_RPC_TIMEOUT_SHORT_REQUESTS: Final[PositiveInt] = int(
```

Comment:
this is only used to remove a task.
so if it is used to stop a dynamic service (which I guess is why there is a 1hour timeour?) this could fail. What will happen then?
and also I doubt that if the frontend calls such an endpoint it will wait for 60 minutes, what will happen then?